src/components/createConnectDecorator.js

```diff
@@ -4,7 +4,7 @@ import shallowEqualScalar from '../utils/shallowEqualScalar';
 export default function createConnectDecorator(React, Connector) {
   const { Component } = React;
 
-  return function connect(select) {
+  return function connect(slicer, actionCreators) {
```

Comment:
Let's change it to `selector`. We call them selectors in the Redux documentation, as well as in [reselect](https://github.com/faassen/reselect) API.

We can rename `slice` to `selectedState`.